packages/suite/src/components/suite/TrafficLightOffset.tsx

```diff
@@ -13,40 +13,39 @@ type Props = {
 
 // See: https://github.com/electron/electron/issues/5678
 // Visible all the time in the app
-const ThinFixForNotBeingAbleToDragWindow = styled.div`
+
+const FixForNotBeingAbleToDragWindow = styled.div`
     -webkit-app-region: drag;
-    height: 16px;
+    pointer-events: none;
+    height: 64px;
     position: fixed;
     z-index: ${zIndices.windowControls};
     top: 0;
     left: 0;
     width: 100%;
 `;
 
-// Visible below content (but visible in the sidebar)
-const ThickFixForNotBeingAbleToDragWindow = styled(ThinFixForNotBeingAbleToDragWindow)`
-    height: 64px;
-    z-index: unset;
-`;
-
 const Container = styled.div<{ $offset: number }>`
     ${({ $offset }) => `padding-top: ${$offset}px;`}
     width: 100%;
     height: 100%;
 `;
 
+export const TrafficLightDraggableWindowHeader = ({ children, isVisible = true }: Props) => {
+    const isMac = isMacOs();
+    const isDesktopApp = isDesktop();
+
+    if (!isVisible || !isMac || !isDesktopApp) return children;
+
+    return <FixForNotBeingAbleToDragWindow />;
```

Comment:
NIT (meaning nitpick, if won't added, it's okay): I would add a comment here explain of why this fix is needed in these conditions, what's the root of it.